PEP 544: Couple final edits (#1084)
python/peps: pep-0544.txt

  c = Other()  # Also OK

Callback protocols
------------------

Protocols can be used to define flexible callback types that are hard
(or even impossible) to express using the ``Callable[...]`` syntax

> specified by PEP 484. For example, variadic, overloaded, and complex generic

Reviewer: This is a sentence fragment
```suggestion
specified by PEP 484, such as variadic, overloaded, and complex generic
```
